Dockerfile

```diff
@@ -1 +1,7 @@
 FROM hone/mruby-cli
+
+RUN groupadd -r mruby -g 1000 && \
+    useradd -d /home/mruby/ -u 1000 -g mruby -r -s /sbin/nologin -c "Docker image mruby user" mruby && \
+    chown -R mruby:mruby /home/mruby/
```

Comment:
> This should go in the upstream container?

Of course, I put it there temporally so you can try it.

> I thought we had issues with this on OS X because how boot2docker deals with permissions

I didn't know that. I'll try on OS X. thanks for the info.